packages/storage-postgres/migrations/0065_personal_bank_customers.sql

```
@@ -0,0 +1,17 @@
+-- #2290 Personal Bank KYC: let bank_customers be owned by a person OR a space.
+-- Existing rows all have space_id set and person_id null, so they satisfy the XOR check.
+ALTER TABLE "bank_customers" ALTER COLUMN "space_id" DROP NOT NULL;
+
+ALTER TABLE "bank_customers" ADD COLUMN IF NOT EXISTS "person_id" integer;
+
+ALTER TABLE "bank_customers"
+  ADD CONSTRAINT "bank_customers_person_id_people_id_fk"
+  FOREIGN KEY ("person_id") REFERENCES "public"."people"("id")
+  ON DELETE no action ON UPDATE no action;
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)